config/routes.rb

```diff
@@ -0,0 +1,3 @@
+UiComponents::Engine.routes.draw do
```

Comment:
A YAML file we control, yes. How is this different form any other code we put on production?

Good point about the spec dir not being packaged. I didn't know.